src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -114,9 +122,13 @@ export const NavBarSchedule: FC<NavBarScheduleProps> = ({ isVisible }) => {
   );
 
   const schedule: ScheduleDay = useMemo(() => {
-    const dayStart = addDays(startOfToday(), selectedDayIndex);
+    const dayStart = addDays(today, selectedDayIndex);
     const daysEvents = relatedVenueEvents
-      .filter(isEventWithinDate(selectedDayIndex === 0 ? Date.now() : dayStart))
+      .filter(
+        isEventWithinDate(
+          selectedDayIndex === 0 && !isScheduleStartDate ? Date.now() : dayStart
+        )
+      )
```

Comment:
What's the intent/reason for this code change here?

Reply:
vscode seems to do that by default — I've since turned off that feature, so additional linebreaks shouldn't be a problem in the future

Reply:
Cool. We use `prettier` for our formatting, so if it has a mode that tells it to use that (which I imagine it should) then that would work even better (though if you don't, it should just happen as part of our git hooks on commit)

Reply:
@margulies Actually.. there is also a logic change here, which I believe is what I was actually asking about here. You adde `!isScheduleStartDate` into the ternary. Why?

Reply:
See my exploration of the logic + suggested solution in https://github.com/sparkletown/sparkle/pull/1419/files#r635726723